docs/src/documentation/02-foundation/07-icons/02-cdn.mdx

```diff
@@ -1,3 +1,24 @@
 ---
 title: CDN
 ---
+
+You can download a PNG file for any Orbit icon at the following address:
+[https://images.kiwi.com/orbit-icons/`{type}`/`{size}`/`{name}`.png](https://images.kiwi.com/orbit-icons/success/32x32/airplane.png).
```

Comment:
The [current documentation](https://orbit.kiwi/foundation/icons/cdn/) does have inline code. This was just a port of that. I'm OK with getting rid of it. I guess was just an additional way of signifying that the link had variables in it (since people very, very often overlook that).

Reply:
Ok, let's keep it, then. We'll adjust the styles for inline code differently inside links.

Reply:
I already took it out 😁